content/learn/about-validated-patterns.adoc

```diff
@@ -0,0 +1,101 @@
+
+
+
+[id="involvement-in-validated-patterns"]
+== Red Hat's involvement in Validated Patterns
+
+Red Hat plays a pivotal role in the development, validation, and promotion of Validated Patterns. As a leader in open-source solutions, Red Hat leverages its extensive expertise to create and maintain these patterns, ensuring they meet the highest standards of quality and reliability. Red Hat’s involvement extends beyond tool provision; it includes continuous updates to align these patterns with the latest technological advancements and industry needs. This ensures that organizations using Validated Patterns are always equipped with the most effective and up-to-date solutions available. Additionally, Red Hat collaborates closely with the community to expand the catalog of Validated Patterns, making these valuable resources accessible to organizations worldwide.
+
+[id="deployment-workflows"]
+== Application deployment workflows
+Effective deployment workflows are crucial for ensuring that applications are deployed consistently and efficiently across various environments. By leveraging OpenShift clusters and automation tools, these workflows can streamline the process, reduce errors, and ensure scalability. Below, we outline the general structure for deploying applications, including edge patterns and GitOps integration.
+
+[id="general-structure"]
+=== General structure
+
+All patterns assume an available OpenShift cluster for deploying applications. If no OpenShift cluster is available, you can use https://console.redhat.com/openshift[cloud.redhat.com]. The documentation uses the `oc` command syntax, but `kubectl` can be used interchangeably. For each deployment, it is assumed that the user is logged into a cluster using the `oc login` command or by exporting the `KUBECONFIG` path.
+
+
+The diagram below outlines the general deployment flow for a data center application. Before proceeding, users must create a fork of the pattern repository to allow for changes to operational elements (such as configurations) and application code. These changes can then be successfully pushed to the forked repository as part of DevOps continuous integration (CI). Clone the repository to your local machine, and push future changes to your fork.
+
+
+image::/images/gitops-datacenter.png[GitOps for Datacenter]
+
+. Make a copy of the values file, for example, `values-global.yaml` or `values-hub.yaml`. These values files specify subscriptions, operators, applications, and other details. Additionally, each Validated Pattern contains a `values-secret` template file, which provides secret values required to successfully install the pattern. Patterns do not require committing secret material to git repositories. It is important to *avoid pushing sensitive information to a public repository accessible to others*. The Validated Patterns framework includes components to facilitate the safe use of secrets.
```

Comment:
Possible edit: "In your fork, if needed, edit the values files, such as `values-global.yaml` or `values-hub.yaml`, to customize or personalize your deployment."

Reply:
Done, as suggested. Thanks!